index.html

```diff
@@ -82,6 +83,7 @@ <h2>ts-root</h2>
 					<ts-button onclick="toggleSlot('sidebar-inner-left', this)">Toggle Inner Left Sidebar</ts-button>
 					<ts-button onclick="toggleSlot('sidebar-inner-right', this)">Toggle Inner Right Sidebar</ts-button>
 					<ts-button onclick="toggleCover('my-cover')">Toggle cover</ts-button>
+					<ts-button onclick="toggleAside('my-aside')">Toggle aside</ts-button>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a 'onKeyPress' attribute to this <ts-button> tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMouseEventWithoutKeyboardEquivalentCheck)